deployment/stacks/api.ts

```diff
@@ -52,7 +52,8 @@ export function createApiComponents(props: CreateApiComponentsProps) {
   Tags.of(scope).add("SERVICE", service);
 
```

Comment:
I wonder if we could solve it [here](https://github.com/Enterprise-CMCS/macpro-mdct-seds/blob/master/deployment/local/prerequisites.ts) and then have a matching values in seds-defaults so that when you run local it finds your local thing as if it were the cloud thing. This is how we solved this same thing for vpnName. I'm fine with a new approach if you prefer this, more than anything I just want it to be consisten.